### ISSUE TITLE
Display Feature Knowledge

### ISSUE THREAD

**Issue by davidk64fnq** [NickMcConnell/FAangband#373]
Is "t"/"T" supposed to do something?

**Comment by davidk64fnq** [comment]
Ah, so you press "t" to select whether LOS/lit etc then press "v" to go in and change it. I see when you press "v" you get the option of "t" but it doesn't seem to do anything? I wonder if the "t" option should only appear once "v" pressed (and be operational there?)

**Comment by backwardsEric** [comment]
That would be an alternative way to have the user interface respond.  If it did behave that way, it would probably be better if selecting a new representation didn't automatically back out to the menu where one pressed 'v'.

If it did remain the way it is, I could see making some slight changes to provide better feedback to the user when 't'/'T' is pressed: change the prompt so it reads something like "torch lighting (change with 't'/'T')" and display a cursor about the glyph/tile (if using a 1x1 multiplier) that corresponds to the selected lighting for the currently selected feature.

That said, I can't say I've used the in-game visual editing besides to check that it seems to be functional.  The handful of times I've changed a tile assignment, I directly edited a preference file.  Feedback from people who've used it heavily for its intended purpose would be useful before changing how it works.
